### DISCUSSION TITLE
Rocket Fuel Theme released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2044] [Show and tell]
Rocket Fuel Theme Features
Extended Control panel:
	Navigation Tree.
	Navigate to any page including deleted pages.
	Admin Dashboard navigation buttons.
	Framework version indicator.
	
Page Management:
	Can change the color of the navigation bar of the theme.
	Can adjust the width of the Panes by updating the width.
		adjust width 50%, 60%,70%, 75%, 80%, 90%, 100%
		
Container Settings:
	Added the ability to add a shadow.
	Added a selection of standard colors:
	Blue, Indigo, Purple, Pink, Red, Orange, Yellow, Green, Teal, Cyan

![image](https://user-images.githubusercontent.com/9447612/157087623-f36706da-3848-4b67-91e5-30ce5a2a9b7d.png)

![image](https://user-images.githubusercontent.com/9447612/157088247-b4b93fbf-512f-4960-be0f-69d8c1f71241.png)

![image](https://user-images.githubusercontent.com/9447612/157087813-469ccad3-0726-4070-8653-00d8b978dcbb.png)

![image](https://user-images.githubusercontent.com/9447612/157088017-3c385271-c9b2-4363-a25a-686ce2bbb614.png)

![image](https://user-images.githubusercontent.com/9447612/157088108-2e752d36-7625-432f-bee0-1ab861eef068.png)

**Comment by curlyfro** [comment]
it looks great!  clean.  i love the UI options.  good job!

**Comment by sbwalker** [comment]
Awesome work @leigh-pointer !! It is great to see how you are leveraging the framework capabilities!!

**Reply by leigh-pointer** [reply]
@sbwalker thanks, and there is more on the near horizon.

**Reply by sbwalker** [reply]
@leigh-pointer I am thinking it might make sense to create a base class for the core ControlPanel component (similar to Login) so that you do not need to duplicate so much logic in your custom control panel?

**Reply by leigh-pointer** [reply]
@sbwalker 

> create a base class for the core ControlPanel component
That sounds good.

I also found some issues in the Menu component (incorrect classes used in the submenu, I will do a PR) so I had to also duplicate that component also.
